scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
```

Comment:
For the semantics of the command I think it would be better to call this option `name` instead of portal:

```bash
$ qiita portal --portal lolercoaster --analyses 1,2,3

$ qiita portal --name lolercoaster --analyses 1,2,3
```

Right?